src/earn/hooks.ts

```diff
@@ -94,3 +95,13 @@ export function useDepositEntrypointInfo({
   }, [asyncExchanges.result])
   return { hasDepositToken, hasTokensOnSameNetwork, hasTokensOnOtherNetworks, canCashIn, exchanges }
 }
+
+export function useEarnPositionUsdAndDepositCryptoValues({ pool }: { pool: EarnPosition }) {
```

Comment:
Nit: `useEarnPositionValues` as a name. If we'd ever have another value returned I could see this name becoming quite long.

Reply:
went with `useEarnPositionBalanceValues` since that makes it a bit more specific to balance but still keeps it generic.